Apps/Database/src/Models/UserProfile.cs

```diff
@@ -98,5 +98,10 @@ public class UserProfile : AuditableEntity
         /// Gets or sets the messaging verifications for this user.
         /// </summary>
         public virtual ICollection<MessagingVerification>? Verifications { get; set; }
+
+        /// <summary>
+        /// Gets or sets the available beta features for this user.
+        /// </summary>
+        public virtual ICollection<BetaFeatureAccess>? BetaFeatureAccessList { get; set; }
```

Comment:
The naming here is strange, which is at least partially caused by this being a collection of the cross-reference table.

There is a way to have a collection for the other entity in the many-to-many relationship (i.e. `BetaFeatureCode`) in place of or in addition to a collection for the cross-reference table.  Some examples are available [here](https://learn.microsoft.com/en-us/ef/core/modeling/relationships/many-to-many#basic-many-to-many) that illustrate how this can be done by convention (using specific entity and property names) or explicitly in the `DbContext.OnModelCreating()` method.

I'd suggest replacing this property with an `ICollection<BetaFeatureCode>? BetaFeatureCodes` and explicitly configuring the relationship in GatewayDbContext to make the naming and usage more straightforward.